File: cp_nim/src/bin/abc392-a-shuffled-equation.nim
# Created by Ayush Biswas at 2026/07/18 19:21
# https://atcoder.jp/contests/adt_easy_20260204_2/tasks/abc392_a
import lib/cpio

# @code begin

import algorithm

var r = newReader()

let a = r[seq[int]].sorted

if a[0] * a[1] == a[2]:
  echo "Yes"
else:
  echo "No"

# @code end

File: cp_rust/src/bin/abc392-a-shuffled-equation.rs
// Created by Ayush Biswas at 2026/07/07 23:05
// https://atcoder.jp/contests/adt_easy_20260204_2/tasks/abc392_a
use cp_lib::*;

// @code begin
use cpio::*;

sol! {
    fn solution(
        (mut a): [usize]
    ) -> Bool {
        a.sort();
        (a[0] * a[1] == a[2]).into()
    }
}
// @code end
